src/server/api/routes/topics.router.js

```diff
@@ -0,0 +1,170 @@
+/* TODO: This is just an example file to illustrate API routing and
+documentation. Can be deleted when the first real route is added. */
+
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const topicsController = require('../controllers/topics.controller');
+
+router.get('/', (req, res, next) => {
+  topicsController
+    .getTopics()
+    .then((result) => res.json(result))
+    .catch(next);
+});
+ 
+   /**
+    * @swagger
+    * /modules/{ID}:
```

Comment:
looks nice! you can change modules to topics.
fx) /topics/{ID}:
     get: summery: Get topic by Id

Reply:
hi Yuka . changes i done .